test/manifest-v7.14.js

```diff
@@ -210,6 +210,66 @@ function getExpectedVector(version) {
           zh: '夏爾',
         },
       },
+      {
+        layer_id: 'shireWithoutIds',
+        created_at: '1532-12-25T18:45:32.389979',
+        attribution: [
+          {
+            label: {
+              en: 'The Silmarillion',
+            },
+          },
+        ],
+        formats: [
+          {
+            type: 'geojson',
+            url: `/files/shire_v2.geo.json`,
+            legacy_default: true,
+          },
+          {
+            type: 'topojson',
+            url: '/files/shire_v2.topo.json',
+            legacy_default: false,
+          },
+        ],
+        fields: [
+          {
+            type: 'id',
+            id: 'wikidata',
+            values: [],
```

Comment:
This is the empty `values` array that the `skipCopy` property should generate.

Reply:
iso empty, I'd consider removing this altogether. The field is optional anyway https://github.com/elastic/kibana/blob/e45209b49f6cfa275d346847e9176e1d8261c22a/x-pack/plugins/maps/public/ems_autosuggest/ems_autosuggest.ts#L28

Reply:
If it's definitely OK to return the object without the values I'll change the code to do so, it's cleaner.